ldi-orchestrator/ldio-connectors/ldio-archive-file-in/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldio/config/LdioArchiveFileInAutoConfig.java

```diff
@@ -16,13 +16,15 @@
 import java.nio.file.Path;
 import java.nio.file.Paths;
 
+import static be.vlaanderen.informatievlaanderen.ldes.ldio.config.PipelineConfig.PIPELINE_NAME;
+
 @Configuration
 public class LdioArchiveFileInAutoConfig {
-
+	private static final String NAME = "be.vlaanderen.informatievlaanderen.ldes.ldio.LdioArchiveFileIn";
```

Comment:
Can't we declare these as public static final Strings in the LdioInput components themselves. In this case that would be LdioArchiveFileIn. Since the name is a property from these classes (it's there full classname after all..).

It would save us a useless constructor parameter for every component..